CoreScriptsRoot/CoreScripts/ContextActionTouch.lua

```diff
@@ -194,6 +194,9 @@ function createButton( actionName, functionInfoTable )
 
 	if buttonScreenGui and buttonScreenGui.Parent == nil then
 		buttonScreenGui.Parent = localPlayer.PlayerGui
+		if not buttonFrame.Parent then
```

Comment:
I will look into this, but regardless; I think we should keep this fix because there's no downside to it and it still works as a solution to the issue if we can't figure out why it's getting parented to nil on respawn.